pkg/internal/sqlsmith/relational.go

```diff
@@ -41,22 +41,28 @@ func makeSchemaTable(s *Smither, refs colRefs, forJoin bool) (tree.TableExpr, co
 	if len(s.tables) == 0 {
 		return makeValuesTable(s, refs, forJoin)
 	}
-	expr, _, _, exprRefs, ok := s.getSchemaTable()
+	expr, _, _, exprRefs, ok := s.getSchemaTable(true /* useIndexHint */)
 	return expr, exprRefs, ok
 }
 
-func (s *Smither) getSchemaTable() (tree.TableExpr, *tree.TableName, *tableRef, colRefs, bool) {
+func (s *Smither) getSchemaTable(
```

Comment:
I would prefer to have two functions that do this instead of having one that is passed a boolean argument. One should be able to call the other and either add or remove the index hint. This should cleanup the call site changes.